bskyembed/test.html

```diff
@@ -0,0 +1,1083 @@
+            </p>
+            &mdash; TJ McIntyre (<a
+              href="https://bsky.app/profile/did:plc:idd5adskstkxy3fzsmer6dtn?ref_src=embed"
+              >@tjmcintyre.com</a
+            >)
+            <a
+              href="https://bsky.app/profile/did:plc:idd5adskstkxy3fzsmer6dtn/post/3lx7j264p7s25?ref_src=embed"
+              >Aug 25, 2025 at 12:46 AM</a
+            >
+          </blockquote>
+          <div>Trusted verifier quoted</div>
+        </div>
+
+        <div class="item">
+          <blockquote
+            class="bluesky-embed"
+            data-bluesky-uri="at://did:plc:p2cp5gopk7mgjegy6wadk3ep/app.bsky.feed.post/3lxciw4rf4c2p"
+            data-bluesky-cid="bafyreicctppfb44vitkn73rx546wvvybs7e7ftkc3dhsz56vm7a6my5fdq"
+            data-bluesky-embed-color-mode="system">
+            <p lang="en">test</p>
+            &mdash; Samuel (<a
+              href="https://bsky.app/profile/did:plc:p2cp5gopk7mgjegy6wadk3ep?ref_src=embed"
+              >@samuel.bsky.team</a
+            >)
+            <a
+              href="https://bsky.app/profile/did:plc:p2cp5gopk7mgjegy6wadk3ep/post/3lxciw4rf4c2p?ref_src=embed"
+              >Aug 26, 2025 at 5:22 AM</a
+            >
+          </blockquote>
+          <div>Verified</div>
+        </div>
+
+        <div class="item">
+          <blockquote
+            class="bluesky-embed"
+            data-bluesky-uri="at://did:plc:p2cp5gopk7mgjegy6wadk3ep/app.bsky.feed.post/3lxdeaccemk2j"
+            data-bluesky-cid="bafyreibmveixwjpcc2a4va2c5wihztsgdlysxvi4bz4xe7ilxe7uvuzn3u"
+            data-bluesky-embed-color-mode="system">
+            <p lang="en">
+              atproto in one post<br /><br /><a
+                href="https://bsky.app/profile/did:plc:p2cp5gopk7mgjegy6wadk3ep/post/3lxdeaccemk2j?ref_src=embed"
+                >[image or embed]</a
+              >
+            </p>
+            &mdash; Samuel (<a
+              href="https://bsky.app/profile/did:plc:p2cp5gopk7mgjegy6wadk3ep?ref_src=embed"
+              >@samuel.bsky.team</a
+            >)
+            <a
+              href="https://bsky.app/profile/did:plc:p2cp5gopk7mgjegy6wadk3ep/post/3lxdeaccemk2j?ref_src=embed"
+              >Aug 26, 2025 at 1:30 PM</a
+            >
+          </blockquote>
+
+          <div>Verified quoted</div>
+        </div>
+      </div>
+    </section>
+
+    <!-- Other languages -->
+    <section>
+      <h1>Other languages</h1>
+      <div class="grid">
+        <div class="item">
+          <div>Japanese</div>
+        </div>
+        <div class="item">
+          <div>Hebrew</div>
+        </div>
+      </div>
+    </section>
+
+    <!-- Misc -->
+    <section>
+      <h1>Misc</h1>
+      <div class="grid">
+        <div class="item">
+          <div>Post from labeler</div>
+        </div>
+        <div class="item">
+          <div>Long name</div>
+        </div>
+        <div class="item">
+          <div>Long handle</div>
+        </div>
+      </div>
+    </section>
```

Comment:
i missed this during my pass, but we can probably get rid of this section. otherwise looks good! was able to run and test locally